### ISSUE TITLE
Accessibility review on DUP Public

### ISSUE THREAD

**Issue by manuji** [bcgov/parks-reso-public#507]
#### Description:
An Accessibility review should be done on DUP Public site to confirm that we follow the WCAG 2.2 AA guidelines and report any issues to the backlog.

#### Acceptance Criteria:

- [ ] Accessibility issues reported in the backlog
- [ ] An accessibility review report is provided

**Comment by manuji** [comment]
Hi @LindsayMacfarlane sorry I missed your questions until now, the exisiting DUP public would take around 20 hours to do a full accessibility review and to log the bug tickets.
